app/api/ada/lib/storage/database/genericBip44/tables.js

```diff
@@ -0,0 +1,391 @@
+// @flow
+
+import {
+  ConceptualWalletSchema,
+  KeySchema,
+} from '../uncategorized/tables';
+import { Type } from 'lovefield';
+import type { lf$schema$Builder } from 'lovefield';
+
+export type Bip44DerivationInsert = {|
+  PublicKeyId: number | null,
+  PrivateKeyId: number | null,
+  IsHardened: boolean,
+|};
+export type Bip44DerivationRow = {|
+  Bip44DerivationId: number, // serial
+  ...Bip44DerivationInsert,
+|};
+export const Bip44DerivationSchema: {
+  name: string,
+  properties: $ObjMapi<Bip44DerivationRow, ToSchemaProp>
+} = {
+  name: 'Bip44Derivation',
+  properties: {
+    Bip44DerivationId: 'Bip44DerivationId',
+    IsHardened: 'IsHardened',
+    PrivateKeyId: 'PrivateKeyId',
+    PublicKeyId: 'PublicKeyId',
+  }
+};
+
+export type Bip44WrapperInsert = {|
+  ConceptualWalletId: number,
+  IsBundled: boolean,
+  SignerLevel: number | null,
```

Comment:
I tried to update/comment the SQL diagram but I couldn't. The **null** for `SignerLevel` is missing from the diagram. Also, I think there is an extra line between Bip44Wrapper and Bip44Derivation tables.

Reply:
There are a few differences with the spec that are required. I will update the spec once this is fully implemented otherwise I'd waste a time update the spec everytime I notice we need a subtle change.